mayastor/src/core/handle.rs

```diff
@@ -240,8 +234,10 @@ impl BdevHandle {
     ) -> Result<usize, CoreError> {
         trace!("Sending nvme_admin {}", nvme_cmd.opc());
         let (s, r) = oneshot::channel::<bool>();
+        // Use the spdk-sys variant spdk_bdev_nvme_admin_passthru that
+        // assumes read commands
         let errno = unsafe {
-            spdk_bdev_nvme_admin_passthru(
+            spdk_bdev_nvme_admin_passthru_read(
```

Comment:
Just a nit, but the trailing _read() to me (which does not mean very much but still) is somewhat misleading. Perhaps, it should be _ro() or _read_only() as now, when reading this, it almost reads as "read something with a admin command" ... but it might be me as non native english person ofcourse :)

Reply:
_ro probably makes more sense so I've renamed it to that.